.github/pull_request_template.md

```diff
@@ -22,6 +22,11 @@ $ make testacc TESTARGS='-run=TestAccXXX'
 Release note for [CHANGELOG](https://github.com/anexia-it/terraform-provider-anxcloud/blob/master/CHANGELOG.md):
 <!--
 If change is not user facing, just write "NONE" in the release-note block below.
+Otherwise use the following format:
+ * <resource-type>[/<resource-name>] - <short description of what has been done>
```

Comment:
maybe add an example here, too, as you did in anexia-it/go-anxcloud#89?

Reply:
True